test/testdata/deployednettemplates/recipes/custom/example/npr/five-relays.txt

```diff
@@ -0,0 +1,20 @@
+R1 R2 116818
```

Comment:
Sorry for not being more up to date on this, this question might be pretty basic:

Looks like we've got `node1 node2 latency-offset (minrtt)`

But then looking at how the file is used in the generation, seems that it throws out all but the first column. I figure that there's future work to use those fields, so I'm wondering what the plan is for handling this format along some edge cases:

* If there is not a row describing a minrtt between two hosts, what does that mean?
 - I assume that means they shouldn't connect (so we'll have to manage their phonebooks), because if it defaulted to a minrtt of `0`, it would invalidate the attempted topology
 
 * If there are hosts implicated only in the `host2` column, current implementation wouldn't discover them. Would that be the desired behavior?
 
 * super minor, but what's the intended strategy when there are multiple conflicting lines? right now it'll skip anything beyond the first reference to the host, but i'll need to take all lines into consideration in the future, yeah?
 
 And then at a general level for this file -- how large might we expect one of these files to be? Thinking N*4, one outbound connection per host. Maybe double that to describe latencies in both directions?